### DISCUSSION TITLE
404 error

### DISCUSSION THREAD

**Discussion by rblackbu** [RedHat-UX/red-hat-design-system#1704] [Documentation feedback]
FYI - seeing a 404 error with https://ux.redhat.com/components/get-started
<img width="1106" alt="Screenshot 2024-07-09 at 11 13 34 AM" src="https://github.com/RedHat-UX/red-hat-design-system/assets/25234746/5510126f-c927-47df-b147-a720e3b36deb">

**Comment by bennypowers** [comment]
where's the referring page?

**Comment by rblackbu** [comment]
I can't find it, Benny. I may have traveled there from an old bookmark in
my browser.
---

Russ Blackburn

Principal User Experience (UX) Designer, Digital Experience

Red Hat <https://www.redhat.com/>
<https://www.redhat.com/>



‪On Wed, Jul 10, 2024 at 9:52 AM ‫Benny Powers - עם ישראל חי!‬‎ <
***@***.***> wrote:‬

**Reply by bennypowers** [reply]
thanks. i'll add it to our redirects table

**Comment by bennypowers** [comment]
#1707
